keyboards/e6_rgb/info.json

```diff
@@ -0,0 +1,22 @@
+{
+  "keyboard_name": "e6",
+  "url": "",
+  "maintainer": "qmk",
+  "bootloader": "",
```

Comment:
This key is deprecated and no longer used:

```suggestion
```

Reply:
ok, this key was removed